src/Analyzers/CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+# Contributing to DurableTask.Analyzers
+
+This document presents guidelines on how to contribute and develop `DurableTask.Analyzers`,
+a series of [Roslyn Analyzers](https://learn.microsoft.com/en-us/visualstudio/code-quality/roslyn-analyzers-overview?view=vs-2022)
+that inspect Durable Task and Azure Functions code for quality, maintainability, code constraints, and other issues. 
+
+## Requirements
+
+- [.NET 6](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+
+## Building
+
+If you are using Visual Studio, you can just open the main [solution file](../../Microsoft.DurableTask.sln)
+and be able to build the [project](./Analyzers.csproj). 
```

Comment:
Can you double check that these links are correct? They weren't working for me.

Reply:
It is working for me 👍 Let's review offline.